### ISSUE TITLE
Broken test: GcMetricsIT

### ISSUE THREAD

**Issue by ctubbsii** [apache/accumulo#1792]
**Test name(s)**
 - org.apache.accumulo.test.functional.GcMetricsIT.gcMetricsPublished

**Describe the failure observed**
```java
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:87)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at org.junit.Assert.assertTrue(Assert.java:53)
	at org.apache.accumulo.test.functional.GcMetricsIT.sanity(GcMetricsIT.java:158)
	at org.apache.accumulo.test.functional.GcMetricsIT.gcMetricsPublished(GcMetricsIT.java:110)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```
STDOUT contained:
```
2020-11-17T06:18:25,372 [miniclusterImpl.MiniAccumuloClusterImpl] INFO : Starting MAC against instance miniInstance and zookeeper(s) localhost:40613.
2020-11-17T06:18:43,920 [functional.GcMetricsIT] DEBUG: test start: 1605593908919, gc start: 1605585884992, gc finished: 1605585889971
```

**Testing Environment:**
 - Version of this project: 2.1.0-SNAPSHOT
 - First commit known to fail (or current commit): 28ba8434334293d4bff9683464b293cd4540c245
 - Jenkins:
```
Executing Maven:  -B -f /home/jenkins/workspace/Accumulo/main/pom.xml -V -e -ntp clean verify -Dstyle.color=always -Dfailsafe.rerunFailingTestsCount=5 -Dsurefire.forkCount=2C
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /usr/local/asfpackages/maven/apache-maven-3.6.3
Java version: 11.0.6, vendor: Oracle Corporation, runtime: /usr/local/asfpackages/java/jdk-11.0.6
Default locale: en_US, platform encoding: ISO-8859-1
OS name: "linux", version: "4.15.0-74-generic", arch: "amd64", family: "unix"
```

**Comment by ctubbsii** [comment]
I thought this was fully addressed in #1745, but it may not have been.

**Comment by ctubbsii** [comment]
I tracked this down to two problems:
1. The test seems to be designed to look for new metrics from new GC cycles. However, what it's actually detecting is new metrics events written to the file, and not new GC cycle metrics.
2. It also seems to be using the relative nanos from the MetricsFileTailer as a demarcation point for the next GC cycle, instead of the previously parsed GC start or finish time.

These seem relatively simple to fix, but I will not be able to fix them immediately. I will take a look next week.
